### PR TITLE
Updates API endpoints for experiments manager commands

### PR DESCRIPTION
**Pull request by simonedavico**
Also, adds a `debug` group of commands and `pygments` for colored output (currently disabled because not extensively tested). Improves the process of testing the client locally without installing.

See the commits for additional informations.

### CODE REVIEW

File: benchflow [hunk 1]
```diff
@@ -14,8 +14,9 @@ IMAGE="benchflow/client:$VERSION"
 # Setup volume mounts for the benchflow client config and context
 VOLUMES="-v $(pwd):$(pwd)"
 
-ENV_EXP="-e EXPERIMENT_MANAGER_ADDRESS=$EXPERIMENT_MANAGER_ADDRESS"
+ENV_EXP="-e EXPERIMENTS_MANAGER_ADDRESS=$EXPERIMENTS_MANAGER_ADDRESS"
```
Comment (VincenzoFerme) [inline benchflow]: Why did you change this file?

File: benchflow.py [hunk 2]
```diff
@@ -2,21 +2,33 @@
...
+from pygments import highlight
+from pygments.formatters import Terminal256Formatter as TermF
+from pygments.lexers import JsonLexer as JsonL
 
-expManagerAddress = os.getenv("EXPERIMENT_MANAGER_ADDRESS")
+
+expManagerAddress = os.getenv("EXPERIMENTS_MANAGER_ADDRESS")
 cassandraIP = os.getenv("CASSANDRA_IP")
 cassandraPORT = os.getenv("CASSANDRA_PORT")
+driversMakerAddress = os.getenv("DRIVERS_MAKER_ADDRESS")
```
Comment (VincenzoFerme) [inline benchflow.py]: Why do you need it?

File: benchflow [hunk 3]
```diff
@@ -14,8 +14,9 @@ IMAGE="benchflow/client:$VERSION"
...
+ENV_DM="-e DRIVERS_MAKER_ADDRESS=$DRIVERS_MAKER_ADDRESS"
```
Comment (VincenzoFerme) [inline benchflow]: Why did you change this file to add the Drivers Maker IP?

File: benchflow.py (hunk 2 above)
Comment (simonedavico) [inline benchflow.py]: I need it to test the `drivers-maker` without having to go through the `experiments-manager`.

Keeping a dedicated version would mean that we would need to update both. Is it really necessary, since we are on a `dev` branch?

Comment (VincenzoFerme) [inline benchflow.py]: No, as long as these changes are documented and defining the variable is not mandatory

File: benchflow (hunk 3 above)
Comment (simonedavico) [inline benchflow]: Same as above, since I implemented a command that uses that address.
